src/main/java/net/aufdemrand/denizen/events/world/LingeringPotionSplashScriptEvent.java

```diff
@@ -0,0 +1,132 @@
+package net.aufdemrand.denizen.events.world;
+
+import net.aufdemrand.denizen.events.BukkitScriptEvent;
+import net.aufdemrand.denizen.objects.dEntity;
+import net.aufdemrand.denizen.objects.dItem;
+import net.aufdemrand.denizen.objects.dLocation;
+import net.aufdemrand.denizen.utilities.DenizenAPI;
+import net.aufdemrand.denizencore.objects.Element;
+import net.aufdemrand.denizencore.objects.dObject;
+import net.aufdemrand.denizencore.scripts.containers.ScriptContainer;
+import net.aufdemrand.denizencore.utilities.CoreUtilities;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.AreaEffectCloud;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.LingeringPotionSplashEvent;
+
+public class LingeringPotionSplashScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // TODO: Fix Event (Maybe spigot)

```

Comment:
I was completely unable to get the event to fire, i.e. the bukkit event wasn't firing. I added debug to test and that's what I came up with